seqr/utils/elasticsearch/es_search.py

```diff
@@ -292,7 +298,7 @@ def _filter_by_genotype(self, inheritance_mode, inheritance_filter, quality_filt
                 self._index_searches[index].append(self._search)
 
     def _filter_compound_hets(self, quality_filters_by_family, annotations_secondary_search):
-        indices = set(self._indices)
+        indices = self._indices
```

Comment:
`set(self._indices)` makes the pre-sorted list out of order and causes un-predictable behavior in `elasticsearch` searching operations.

Reply:
it is crucial to deduplicate here which is what casting to a set does, so this is an unacceptable changed

Reply:
The order is also important. Since deduplicate is crucial, I will try to sort the result of the casting.